test/presubmit-config-check.sh

```diff
@@ -0,0 +1,84 @@
+#!/usr/bin/env bash
+
+# Copyright 2020 The Knative Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+source $(dirname $0)/../scripts/library.sh
+
+# Run diff on the two configuration files.
+# Parameters: $1 - the regenerated temp file.
+#             $2 - the existing file.
+function diff_config_files() {
+  diff --ignore-matching-lines="^# Copyright " "$1" "$2"
+}
+
+# Run diff on the Prow configuration files.
+# Parameters: $1 - the regenerated Prow core config file.
+#             $2 - the regenerated Prow plugin config file.
+#             $3 - the regenerated Prow job config file.
+#             $4 - the environent, can be prow or prow-staging.
+function diff_prow_config_files() {
+  local prow_config="$1"
+  local prow_plugin_config="$2"
+  local prow_job_config="$3"
+  local prow_env="$4"
+  diff_config_files "${prow_config}" "config/${prow_env}/core/config.yaml"
+  diff_config_files "${prow_plugin_config}" "config/${prow_env}/core/plugins.yaml"
+  diff_config_files "${prow_job_config}" "config/${prow_env}/jobs/config.yaml"
+}
+
+set -e
+
+trap 'echo "Please rerun \`make -C config/prow config\`."' ERR
```

Comment:
Could you add "--- FAIL: " to the beginning of the output for this and all trap?

Reply:
Done.